doc/daal4py/model-builders.rst

```diff
@@ -19,64 +19,69 @@
 ###############################################
 Model Builders for the Gradient Boosting Frameworks
 ###############################################
+
+.. include:: note.rst
+
 Introduction
 ------------------
 Gradient boosting on decision trees is one of the most accurate and efficient 
 machine learning algorithms for classification and regression. 
-The most popular implementations of it are the XGBoost*, 
-LightGBM*, and CatBoost* frameworks.
+The most popular implementations of it are: 
+
+* XGBoost*
+* LightGBM*
+* CatBoost*
+
 daal4py Model Builders deliver the accelerated
 models inference of those frameworks. The inference is performed by the oneDAL GBT implementation tuned 
 for the best performance on the Intel(R) Architecture. 
 
 Conversion
 ---------
-The first step is to convert already trained model. There are similar 
-APIs for different frameworks. 
+The first step is to convert already trained model. The
+API usage for different frameworks is the same:
+
 XGBoost::
 
   import daal4py as d4p
-  d4p_model = d4p.get_gbt_model_from_xgboost(xgb_model)
+  daal_model = d4p.mb.convert_model(xgb_model)
```

Comment:
```suggestion
  d4p_model = d4p.mb.convert_model(xgb_model)
```
I think this would be safe to talk about daal4py model and not daal itself. There would be some sort of renaming for modelbuilders, but for sure this shouldn't be daal model

Reply:
thanks, fixed